iroha/src/genesis.rs

```diff
@@ -242,6 +243,9 @@ pub mod config {
                             format!("Failed to parse Private Key of genesis account: {}", e)
                         })?;
             }
+
+            self.genesis_block_path = env::var(GENESIS_BLOCK_PATH).ok();
+
```

Comment:
Genesis Block should not be loaded from environment its CLI parameter only. It is done so that only the peer that wants to submit the genesis block, will specifically mention it at the start.